src/internal/connector/onedrive/collection.go

```diff
@@ -339,16 +338,25 @@ func (oc *Collection) getDriveItemContent(
 	// check for errors following retries
 	if err != nil {
 		if clues.HasLabel(err, graph.LabelsMalware) || (item != nil && item.GetMalware() != nil) {
-			logger.Ctx(ctx).With("error", err.Error(), "malware", true).Error("downloading item")
-			el.AddSkip(fault.FileSkip(fault.SkipMalware, itemID, itemName, graph.MalwareInfo(item)))
-		} else {
-			logger.Ctx(ctx).With("error", err.Error()).Error("downloading item")
-			el.AddRecoverable(clues.Stack(err).WithClues(ctx).Label(fault.LabelForceNoBackupCreation))
+			logger.Ctx(ctx).With("error", err.Error(), "reason", "malware").Error("downloading item")
```

Comment:
We have a QoL for logging error details now!

Also, nit and not a big deal, I wouldn't call skips an error.  It's a well-recognized and well-handled condition.  Let's keep with info.
```suggestion
			logger.CtxErr(ctx, err).With("skip_reason", "malware").Info("downloading item")
```